src/main/augurNodeServer.js

```diff
@@ -247,6 +247,7 @@ AugurNodeServer.prototype.shutDownServer = function () {
     if (this.augurNodeController == null || !this.augurNodeController.isRunning()) return
     log.info('Stopping Augur Node Server')
     this.augurNodeController.shutdown()
+    this.window.webContents.send('onServerDisconnected', {})
   } catch (err) {
```

Comment:
Might need to add this to when an error is thrown. Could check is is running on exception.